assets/dev/scss/editor/_global.scss

```diff
@@ -166,3 +165,31 @@
 i.eicon-tilted {
 	transform: rotate(90deg);
 }
+
+.elementor-control .elementor-control-content.elementor-control-media-size {
+	margin-top: 15px;
+}
+
+.elementor-control-media-size .elementor-control-field {
+	padding: 0px;
+}
+
+.elementor-control-media-size .elementor-control-responsive-switchers {
+	display: none;
```

Comment:
why?

Reply:
There is no need to show twice the breakpoint switcher in the same control. 
I extend the media control, not added a new one.

Reply:
In the near future, we will need to support several such breakpoint support buttons in one control, so I prefer not to make changes to this library.